lib/jambots/bot.rb

```diff
@@ -126,5 +124,24 @@ def load_bot_options(bot_yml_path)
 
       YAML.safe_load(File.read(bot_yml_path), permitted_classes: [Symbol], symbolize_names: true)
     end
+
+    def handle_error(response)
+      if response.dig("error", "code") == "invalid_api_key"
+        <<~HEREDOC
+          Invalid OpenAI API key. Please set the OPENAI_API_KEY environment variable to your OpenAI API key.
+          You can find your API key at https://beta.openai.com/account/api-keys.
+        HEREDOC
+      elsif response.dig("error", "code") == "max_tokens"
+        <<~HEREDOC
+          The chat is too long and exceeds the maximum number of tokens. The chat is very long and exceeds the maximum number of tokens.
+          Check the limitations of the model "#{model}" https://platform.openai.com/docs/models/overview
+        HEREDOC
+      else
+        <<~HEREDOC
+          OpenAI error - #{response["error"]["message"]}.
+          #{response}
+        HEREDOC
+      end
+    end
```

Comment:
Should this be an error renderer? Makes sense to use the same concept

Reply:
I created an issue to change it https://github.com/artero/jambots/issues/16